packages/manager/modules/bm-server-components/src/technical-details/technical-details.controller.js

```diff
@@ -35,9 +35,12 @@ export default class BmServerComponentsTechnicalDetailsController {
 
   isRamUpgradable() {
     return (
-      this.upgradeWithTicketAvailable &&
-      this.technicalDetails.memory?.upgradable?.length &&
-      !this.upgradeTask
+      (this.showServiceActions &&
+        this.upgradeWithTicketAvailable &&
+        this.technicalDetails.memory?.upgradable?.length &&
+        !this.upgradeTask) ||
+      (this.showServiceActions &&
+        this.upgradeTask.shouldScheduleUpgrade(this.UPGRADE_TYPE.RAM))
```

Comment:
**nitpick:**  Can be simplified.
```suggestion
      this.showServiceActions && (
        this.upgradeWithTicketAvailable &&
        this.technicalDetails.memory?.upgradable?.length &&
        !this.upgradeTask) ||
        this.upgradeTask.shouldScheduleUpgrade(this.UPGRADE_TYPE.RAM)
```

Reply:
Yes its possible but I kept like this to have better readibility.